LoggerConfig/LoggerImplementation/AutoLog.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Discord;
+using Serilog.Events;
+using Serilog;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace QuickEdit.LoggerConfig.LoggerImplementation;
+
+public class AutoLog
```

Comment:
I think `AutoLog` sounds a bit weird, but I honestly can't think of anything else except for like `DiscordLogger`, which would kind of sound like it's logging actual messages from discord, which would sound a bit like the bot is "spying"

The namespace makes `AutoLog` sound better though, so I think it could be left like this